src/components/text-editor/prosemirror-adapter/prosemirror-adapter.scss

```diff
@@ -11,6 +11,7 @@
     div#editor {
         order: 2;
         height: 100%;
+        flex-grow: 1;
```

Comment:
<img width="451" alt="Screenshot 2024-06-11 at 11 46 19" src="https://github.com/Lundalogik/lime-elements/assets/35954987/6c41adc7-4ce3-4720-bcb3-b79c3c3bd292">

Reply:
There doesn't seem to be any example in the docs that show the difference this makes, but I'm perfectly fine with taking your word for it 👍 😊